src/L.Realtime.js

```diff
@@ -10,11 +10,12 @@ L.Realtime = L.Layer.extend({
         updateFeature: function(f, oldLayer) {
             if (!oldLayer) { return; }
 
-            if (f.geometry.type === 'Point') {
+            if (oldLayer.setLatLng) {
                 var c = f.geometry.coordinates;
                 oldLayer.setLatLng([c[1], c[0]]);
                 return oldLayer;
             }
+            return oldLayer;
```

Comment:
This will _always_ return the old layer, even though it wasn't modified in any way. That will make polygons, linestrings, etc. essentially static, since they will never reflect changes to their coordinates etc.